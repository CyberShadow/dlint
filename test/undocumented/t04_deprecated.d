// Deprecated symbols are exempt from being documented.

void bad1() {}
deprecated void good2() {}
deprecated struct good3
{
	void good4() {}
}
