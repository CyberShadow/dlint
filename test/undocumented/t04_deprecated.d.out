undocumented/t04_deprecated.d(3,6): Undocumented public declaration: FuncDeclaration `bad1`
